src/nvim/log.c

```diff
@@ -69,6 +69,12 @@ static bool log_path_init(void)
       || log_file_path[0] == '\0'
       || os_isdir((char_u *)log_file_path)
       || !log_try_create(log_file_path)) {
+    // Make kXDGCacheHome if it does not exist.
+    char *cachehome = get_xdg_home(kXDGCacheHome);
+    if (!os_isdir((char_u *)cachehome)) {
+      os_mkdir((char *)cachehome, 0700);
```

Comment:
This should probably be `os_mkdir_recurse()`, since we don't know how much (if any) of XDG_CACHE_HOME will already exist.

Reply:
I thought about that, I'll go ahead and change it then